core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties

```diff
@@ -0,0 +1,33 @@
+title=Manage Legacy API Token usage
+contextMessage=The following users are using Legacy API Token. \
```

Comment:
nit: I would reword this to: "The following users have a legacy API token. Because legacy tokens are stored in a recoverable format, we recommend migrating to the new API token system. Additionally, in previous version of Jenkins, API tokens were automatically created for every new user. Often, users did not use the token, resulting in a larger attack surface than necessary. Automatic token generation must now be enabled explicitly."